app/src/main/java/src/utilities/Menu.java

```diff
@@ -116,23 +136,38 @@ public void showMovementMenu(final Game game, final GameBoard gameBoard, final S
             int option = scanner.nextInt();
             switch (option) {
                 case OPTION_1:
-                    System.out.println("Moved to right");
-                    shape.moveRight();
-                    shape.moveDown();
+                    if (game.checkCollision(shape, gameBoard, 2 + 1)) {
+                        collision = true;
+                        gameBoard.setGameBoardArray(shape.getxPosition(), shape.getyPosition(), shape.getContainer());
+                    } else {
+                        System.out.println("Moved to right");
+                        shape.moveRight();
+                        shape.moveDown();
+                    }
                     break;
                 case OPTION_2:
-                    System.out.println("Moved to left");
-                    shape.moveLeft();
-                    shape.moveDown();
+                    if (game.checkCollision(shape, gameBoard, 2 + 1)) {
+                        collision = true;
+                        gameBoard.setGameBoardArray(shape.getxPosition(), shape.getyPosition(), shape.getContainer());
```

Comment:
This method should receive a shape instead of an array

Reply:
Thanks, I will change it

Reply:
It is done

Reply:
ok jessi, good job